Code/Engine/Core/Utils/Blackboard.h

```diff
@@ -180,8 +186,9 @@ class EZ_CORE_DLL ezBlackboard : public ezRefCounted
   static ezBlackboard* Reflection_GetOrCreateGlobal(const ezHashedString& sName);
   static ezBlackboard* Reflection_FindGlobal(ezTempHashedString sName);
   void Reflection_RegisterEntry(const ezHashedString& sName, const ezVariant& initialValue, bool bSave, bool bOnChangeEvent);
-  bool Reflection_SetEntryValue(ezTempHashedString sName, const ezVariant& value);
+  void Reflection_SetEntryValue(ezStringView sName, const ezVariant& value);
```

Comment:
No need for this reflection wrapper anymore. This was only necessary because of the ezResult return value.